Parse/PFObject.h

```diff
@@ -569,7 +569,7 @@ NS_REQUIRES_PROPERTY_DEFINITIONS
 
  @returns The task that encapsulates the work being done.
  */
-- (BFTask *)fetchInBackground;
+- (BFTask PF_GENERIC(__kindof PFObject *)*)fetchInBackground;
```

Comment:
We cannot. Instancetype can only be used as the **complete** return type of a method (e.g. `const instancetype` is also invalid).